adapters/mobfoxpb/mobfoxpb.go

```diff
@@ -0,0 +1,131 @@
+package mobfoxpb
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/buger/jsonparser"
+	"github.com/mxmCherry/openrtb"
+	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/config"
+	"github.com/prebid/prebid-server/errortypes"
+	"github.com/prebid/prebid-server/openrtb_ext"
+)
+
+// MobfoxpbAdapter adapter structure
+type MobfoxpbAdapter struct {
```

Comment:
We prefer the name `adapter` here. No need to export it since it's not referenced directly outside of this package. No need to duplicate the mobfox name.

Reply:
done